backend/minors/MAT.yaml

```diff
@@ -0,0 +1,96 @@
+---
+type: Minor
+name: Mathematics
+code: MAT
+degree:
+urls:
+- https://www.math.princeton.edu/undergraduate/minor-mathematics
+- https://ua.princeton.edu/fields-study/departmental-majors-degree-bachelor-arts/mathematics
+description: The goal of the minor in mathematics is to allow students 
+to explore contemporary mathematics, either as a topic for its own sake 
+or to gain a deeper knowledge of a specific area most relevant for the 
+students’ work in their major field of concentration. In consultation 
+with the minor program director of the mathematics department, the 
+student should develop a plan that complements their work in their 
+chosen field of concentration. The plan can be to gain a broad knowledge 
+of mathematics, in which case we recommend courses in algebra, complex 
+analysis and real analysis. Alternatively, students with a strong 
+interest in a more specialized area of mathematics can choose a 
+sequence of courses in that field. The student should contact the minor 
+program director of the mathematics department during the spring of 
+sophomore year to have a course plan approved. Later admission to the 
+Program is allowed during the junior year, though this offers less 
+flexibility in the choice of the required courses. 
+excluded_major: 
+- MAT
+contacts:
+- type: Director of Undergraduate Studies
+  name: János Kollár
+  email: kollar@math.princeton.edu
+req_list:
+- name: Prerequisites
+  max_counted: 1
+  min_needed: 3
+  explanation: The recommended preparation for the minor is the same as
+  for the major in math; knowledge of calculus (both single and several
+  variable), linear algebra and familiarity with proofs. With the prior 
+  approval of the minor program director, students may count one course 
+  taken in another department toward the minor prerequisites. These 
+  courses should be completed by the end of sophomore year.
+  double_counting_allowed: true
+  completed_by_semester: 4
+  req_list:
+  - name: Several Variable Calculus
+    max_counted: 1
```

Comment:
Let's add `double counting true` for these reqs since "217 and 218 each cover two of the recommended preparations"